src/SFA.DAS.AssessorService.Application.Api/Services/StandardService.cs

```diff
@@ -57,6 +57,11 @@ public async Task<IEnumerable<Standard>> GetStandardVersionsByIFateReferenceNumb
             try
             {
                 standards = await _standardRepository.GetStandardVersionsByIFateReferenceNumber(iFateReferenceNumber);
+
+                if (standards == null || !standards.Any())
+                {
+                    _logger.LogInformation($"Unable to find standard for IFATE reference number: {iFateReferenceNumber}");
```

Comment:
There messages should be consistently using the parameter name passed to the method